components/frontend-react/src/components/Appbar.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//MUI imports
+import {makeStyles} from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Avatar from '@material-ui/core/Avatar';
+import TextField from '@material-ui/core/TextField';
+
+//Logo import
+import logo from '../logo.png';
+
+const colours = {
+  somewhat_brown: '#625042',
+  white: '#fff'
+}
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  grow: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginLeft: -12,
+    marginRight: 20,
+  },
+  avatar: {
+    margin: 10,
+  },
+  bigAvatar: {
+    margin: 10,
+    width: 60,
+    height: 60,
+  },
+}));
+
+const BassaAppBar = (props) => {
+  const classes = useStyles();
+
+  if (!props.isloggedIn) {
```

Comment:
Some of the components repeats. Come with the logic inside the component return rather repeating.  Just a suggestion

Reply:
The intention for doing that was because I had two display logo with two different specs. If you check down the code, you will see that I have used one for logged-in and logged-out view